ZFitter/src/auxFunctions.cc

```diff
@@ -9,6 +9,8 @@ std::string energyBranchNameFromInvMassName(std::string invMass_var)
 	else if(invMass_var == "invMass_5x5SC") energyBranchName = "energy_5x5SC";
 	else if(invMass_var == "invMass") energyBranchName = "energyEle";
 	else if(invMass_var == "invMass_rawSC") energyBranchName = "rawEnergySCEle";
+	else if(invMass_var == "invMass_SC_corr") energyBranchName = "energySCEle_corr";
```

Comment:
They won't exist anymore in the next production, they still exist in the latest ntuples from miniAOD. I suggest to remove these two branches later on